### ISSUE TITLE
"Approval" section in docs not applicable for Gitlab upstreams

### ISSUE THREAD

**Issue by jrybar-rh** [packit/packit-service#1707]
The section [2. Approval](https://packit.dev/docs/guide/#2-approval) in Packit Onboarding Guide only works for Github upstreams. 
Gitlab users need to be approved manually every time.

**Comment by TomasTomecek** [comment]
This is correct, I had to manually approve the namespace - the automated approval workflow is not triggered on GitLab right now.

**Comment by lachmanfrantisek** [comment]
Slightly related to #1708.

What needs to be done on our side:
* [ ] create a new `notifications` repository on each forge we want to support
* [ ] make Packit create a new issue in that repository if a user is not allowed and the issue does not exist. (Currently, the issue is created only after the GitHub app installation.)
* [ ] update the code to accept a list of projects, not a single one (ideally, make it configurable):
  * https://github.com/packit/packit-service/blob/da94cc9b511843d73e9b94d20677f0b1d1426bf8/packit_service/worker/handlers/forges.py#L123-L124
  * https://github.com/packit/packit-service/blob/da94cc9b511843d73e9b94d20677f0b1d1426bf8/packit_service/worker/handlers/forges.py#L148-L159
  * (Looking at the code, any `forge.dot.something/packit/notifications` should work now already..;)
* [ ] update docs

**Comment by TomasTomecek** [comment]
We need to scope this properly as there are many paths to take.

**Comment by lachmanfrantisek** [comment]
To make it clear: ideally, we will get rid of the per-installation requirements and do a per-job check (so e.g., we will not require FAS for some types of jobs).

As a result of that work, we don't need to support GitLab in the approval process (this issue) because it will not be needed at all.

**Comment by lachmanfrantisek** [comment]
@jrybar-rh Hello, this issue (and #1708 as well) will not be needed once we finish our https://github.com/packit/packit-service/issues/1850 epic.

With that, I am closing this. Feel free to watch the epic and/or we can discuss it at the weekly sync meeting.
